tasks.py

```diff
@@ -0,0 +1,646 @@
+
+        else:
+
+            run('echo "===>{} {}"'.format(package, MESSAGE_FAILED))
+
+    # installing linuxbrew
+    if PLATFORM == "Linux":
+        run(
+            'sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)",
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "test -d /home/linuxbrew/.linuxbrew && eval \$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)",
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'echo "eval $($(brew --prefix)/bin/brew shellenv)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "sudo apt-get install build-essential curl git", hide=not verbose, warn=True
+        )
+        if run("command -v brew", hide=not verbose, warn=True).ok:
+            run('echo "===>brew {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>brew {}"'.format(MESSAGE_FAILED))
+
+    if PLATFORM == "Darwin":
+        if run("command -v cask ", hide=not verbose, warn=True).ok:
+            run('echo "===>Cask {}"'.format(MESSAGE_ALREADY_INSTALLED))
+        else:
+            if run("brew install cask").ok:
+                run('echo "===>Cask {}"'.format(MESSAGE_OK))
+            else:
+                run('echo "===>Cask {}"'.format(MESSAGE_FAILED))
+
+    # creating a local setting file
+    if run('cd project/settings && touch local.py && echo "DEBUG=True" > local.py').ok:
+        run('echo "===>Successfully configured settings for local development"')
+
+
+@task
+def install_dependencies(c, verbose=False):
+    """
+
+    Install all the dependencies of the API
+    usage: invoke install-dependencies or invoke install-dependencies --verbose
+
+    """
+
+    run("echo '***INSTALLING ALL DEPENDENCIES***'")
+    packages = ("", "django-sslserver")
+    for package in packages:
+
+        if run("pipenv install {}".format(package), hide=not verbose, warn=True).ok:
+            run('echo "===>{} {}"'.format(package, MESSAGE_OK))
+        else:
+            run('echo "===>{} {}"'.format(package, MESSAGE_FAILED))
+
+
+@task
+def pygraphviz(c, verbose=False):
+    """
+
+    Install graphyviz & pygraphviz 
+    usage: invoke pygraphviz or invoke pygraphviz --verbose
+
+    """
+
+    if PLATFORM == "Linux":
+        # Installing graphviz
+        if run(
+            "sudo apt-get update && sudo apt-get install graphviz",
+            hide=not verbose,
+            warn=True,
+        ).ok:
+            run('echo "Graphyviz {}"'.format(MESSAGE_OK))
```

Comment:
s/Graphyviz/Graphviz/g

Reply:
I assume you meant I should write correct "Graphyviz" with "graphviz."

Reply:
https://vim.fandom.com/wiki/Search_and_replace

s/foo/bar/g --> "replace all instances of foo with bar"

Reply:
Got it!